source/common/conn_pool/conn_pool_base.h

```diff
@@ -253,6 +250,9 @@ class ConnPoolImplBase : protected Logger::Loggable<Logger::Id::pool> {
   // The number of streams that can be immediately dispatched
   // if all CONNECTING connections become connected.
   uint32_t connecting_stream_capacity_{0};
+
+  void onUpstreamReady();
```

Comment:
nit: Usual ordering of elements in the private section according to style guide is:
- classes/structs
- functions
- data members